hasura/graphql-engine
# console: append "IF NOT EXISTS" for adding and "IF EXISTS" for removing of columns (fix #5284)

console: append "IF NOT EXISTS" for adding and "IF EXISTS" for removing of columns (fix #5284)

### Description

Added
- "IF NOT EXISTS" to ALTER TABLE ADD COLUMN in up and down migrations
- "IF EXISTS" to ALTER TABLE DROP COLUMN in up and down migrations
This helps to avoid potential problem of failing a migration that adds or removes columns twice.

### Changelog


### Affected components


### Related Issues
#5284

### Steps to test and verify
Unfortunately I wasn't able to make cypress catch an ajax call to /apis/migrate endpoint to verify the payload. To verify the fix manual testing is required.
